src/components/structures/PipContainer.tsx

```diff
@@ -305,6 +305,7 @@ class PipContainerInner extends React.Component<IProps, IState> {
             const call = this.state.primaryCall;
             pipContent.push(({ onStartMoving, onResize }) => (
                 <LegacyCallView
+                    key="call-view"
```

Comment:
any particular reason these `key`s are now needed?

Reply:
They fix React warnings from this area of code, they were always needed

https://github.com/matrix-org/matrix-react-sdk/pull/12094/commits/977216cb90ddcde4098616570047d9f31798267e